webclient/index.html

```diff
@@ -666,7 +668,7 @@
 
     function validatePrompt(prompt) {
         const minLength = 3,
-            maxLength = 500;
+            maxLength = promptLengthMaxBaseUser;
```

Comment:
Ok, we can set max length as constant for base user before authentication implementation. Are we going to set the maxLength for the user after login or are we just going to set an "if else" statement here to change length?

Reply:
Hey! Thanks for the comment. Indeed, validation branching is expected to separate offering by users categories.

The PR contains the adjustment to match the server logic for quick feedback on the client side. It does not include full-fledged solution yet.